controls/nist_ocp4.yml

```diff
@@ -13204,7 +13204,8 @@ controls:
     [2] https://docs.openshift.com/container-platform/4.7/applications/quotas/quotas-setting-per-project.html
     [3] https://docs.openshift.com/container-platform/4.7/applications/quotas/quotas-setting-across-multiple-projects.html
     [4] https://docs.openshift.com/container-platform/4.7/nodes/pods/nodes-pods-configuring.html#nodes-pods-configuring-bandwidth_nodes-pods-configuring
-  rules: []
+  rules: 
+  - routes_rate_limit
```

Comment:
could you please change the status as well? There is a description of the statuses here: https://complianceascode.readthedocs.io/en/latest/manual/developer/03_creating_content.html#reporting-status